doc/Extrusion-rate-smoothing.md

```diff
@@ -0,0 +1,154 @@
+<h1>Extrusion rate smoothing (known as pressure equalizer in Prusa Slicer)</h1>

```

Comment:
I'm thinking would it be better to move `(known as pressure equalizer in Prusa Slicer)` to summary paragraph below?